### PR TITLE
Added a file to indicate how to customize gluegun ATM.

### PR DESCRIPTION
**Pull request by deekoder**

### CODE REVIEW

File: customize.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+$dropdown-box-shadow: 0 2px 10px 0 rgba($black,.05);
+$dropdown-border-color: #efefef;
+$dropdown-item-hover-bg: #f7f7f7;
+$dropdown-item-color: $text-muted-color;
+
+
+// Footer text color
+$footer-text-color: $text-muted-color;
+
+```
```
Comment (kaankabalak) [inline customize.md]: I think we would also need to give users instructions on how to generate the gluegun.css and gluegun.min.css files from the .scss files, or let them know that they have to generate these files, as they won't be able to see the changes by only editing the .scss files

Comment (deekoder) [reply customize.md]: yes agreed.

Comment (deekoder) [reply customize.md]: pending checking in the packages.json and gulp file we can add instructions on how to spit out the css files from modifying variables.scss

**Review by kaankabalak — APPROVED** [review]

**Comment by rushenn** [comment]
@deekoder I think we should not write the SCSS variables here as some of the variables such as `$text-muted-color`, `$muted-bg` may confuse the user without any given description. I think the line 

> Fork gluegun repository and edit the file scss/inc/_variables.scss to create your own themes.

Is suffice.

**Pull request closed without merging**